frame/test_frame.py

```diff
@@ -0,0 +1,18 @@
+from frame_test_case import FrameTestCase
+import time
+
+short_wait_time = 1
```

Comment:
We probably want a longer wait here. A second is likely to not be enough for whatever can go wrong, to go wrong, across all the hardware we're testing on.

Reply:
I bumped to 3, not sure if that's enough